import_flow_events.py

```diff
@@ -32,6 +32,13 @@
 EVENTS_PREFIX = "fxa-flow/data/"
 EVENTS_FILE_URL = "s3://" + EVENTS_BUCKET + "/" + EVENTS_PREFIX + "flow-{day}.csv"
 
+# The CSV format has changed over time. This config controls the
+# column names, definitions and deploy dates for those changes.
+ADDITIONAL_COLUMNS = (
+    {"name":"locale", "definition":"VARCHAR(40)", "deploy_date":"2017-02-21"},
+    {"name":"uid", "definition":"VARCHAR(64)", "deploy_date":"2017-02-21"},
```

Comment:
As per https://bugzilla.mozilla.org/show_bug.cgi?id=1339405#c1, the extra fields begin part-way through the 23rd. I updated the dates here to match.